src/AddTodoForm.js

```diff
@@ -1,19 +1,20 @@
 import React, { useState } from 'react';
+import InputWithLabel from './InputWithLabel';
 const AddTodoForm=({onAddTodo})=>{
 
 // State
 const [todoTitle, setTodoTitle] = useState(); 
 
-let handleTitleChange=(e)=>{
+const handleTitleChange=(e)=>{
      
-     let newTodoTitle=e.target.value;  
+    const newTodoTitle=e.target.value;   
      setTodoTitle(newTodoTitle);
      console.log(todoTitle);
 }
 
-let handleAddTodo=(e)=>{
+const handleAddTodo=(e)=>{
     e.preventDefault();
```

Comment:
If you remove this, the `input` will always get focus after you add a todoItem because the page will actually submit now

Reply:
Heads up!! Doing this actually created a bug for me in the exercise 1.7. Soop dont do this haha. My fault!!